README.md

```diff
@@ -42,6 +39,39 @@ use_frameworks!
 
 Then run `pod install` with CocoaPods 0.36 or newer.
 
+## Adding Runes as a Framework Dependency ##
+
+Runes can also be used as a low-overhead dependency for defining these
+operators for your own types. After linking to `Runes.swift`, you will be able
+to define implementations for your types, and your users will be able to just
+import your framework to use these operators.
+
+### [Carthage] ###
+
+```
+github "thoughtbot/runes"
```

Comment:
this might be old but might as well change it while we're in here `thoughtbot/Runes`